helm/default-apps-azure/values.yaml

```diff
@@ -3,21 +3,33 @@ organization: ""
 
 # filled by cluster-apps-operator
 # via <cluster-name>-cluster-values configmap
+baseDomain: ""
 cluster:
   private: false
+subscriptionID: ""
```

Comment:
`baseDomain` is provider-independent and it is already part of cluster-values cm. I will add `subscriptionID` and `identityClientID` to the cluster-values configmap so that we can use them here. 

What is the best place to put them?  `providerSpecific`? CC: @marians @nprokopic